pom.xml

```diff
@@ -513,6 +517,7 @@
                 <version>${license-maven-plugin.version}</version>
                 <configuration>
                     <header>com/mycila/maven/plugin/license/templates/MIT.txt</header>
+                    <aggregate>true</aggregate>
```

Comment:
What is the function of this aggregate flag?

Reply:
I thought I reverted that, guess not, let me commit that.